src/Bookings.js

```diff
@@ -1,18 +1,18 @@
 import React from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
```

Comment:
Small stylistic thing, `PascalCase` is often reserved for React components, with normal JS variables being `camelCase` - you're importing a JSON file here so this should be `camelCase` as it is not a React component (it can look confusing as people will assume `FakeBookings` is a React component). This should be `import fakeBookings from "./data/fakeBookings.json"`